test/acceptance/integration/cb_mdw/cb_mdw.feature

```diff
@@ -279,11 +294,12 @@ Feature: Context broker middleware
     | json   | /v1/registry/contextAvailabilitySubscriptions/subscriptionId | DELETE |
 
   Scenario Outline: Parameters-Query in cb urls with operation in the payload
-    Given a domain in KEYSTONE
-    And a user in the domain
-    And a project in the user
-    And url with "/v1/updateContext?details=on&limit=15&offset=0" and the actionType attribute "<attribute>"
-    When a context broker "POST" petition is asked to PEP with "<format>" format
+    Given a KEYSTONE CONFIGURATION with all roles in the same project
+    And set the request URL with the path "/v1/updateContext?details=on&limit=15&offset=0"
```

Comment:
No, the purpose of the test is check PEP sends the parms you send, it doesn't matter which (the final target is not a context broker, it is a mock)
NTC